core/src/collections.rs

```diff
@@ -2,8 +2,34 @@ use crate::err::Result;
 use crate::stmt::MongoStatement;
 use crate::{conn::MongoConnection, Error};
 use bson::{doc, Bson, Document};
+use constants::{SQL_ALL_CATALOGS, SQL_ALL_TABLE_TYPES};
+use lazy_static::lazy_static;
 use mongodb::results::CollectionSpecification;
 use mongodb::sync::Cursor;
+use regex::{RegexSet, RegexSetBuilder};
+
+const COLLECTION: &str = "collection";
+const TABLE: &str = "table";
+const VIEW: &str = "view";
+const TIMESERIES: &str = "timeseries";
```

Comment:
Yes, the `timeseries` collections  are queried like normal collections.  
I made the change to add timeseries and collections to the filter only when the table type filter has `TABLE` and removed the check for a `timeseries` entry in the string.  
This will return the timeseries collections when there is no filter on the type, similar to the JDBC driver.

Reply:
Perfect, thanks for looking into this :)